src/neo/Ledger/Blockchain.cs

```diff
@@ -359,6 +357,8 @@ private VerifyResult OnNewBlock(Block block)
                         Self.Tell(unverifiedBlock, ActorRefs.NoSender);
```

Comment:
This is not parallel, right?
In this sense, we were telling heights that were cached?

Reply:
previously was before persists, now after it